nilearn/decoding/searchlight.py

```diff
@@ -510,6 +521,29 @@ def transform(self, imgs):
 
         return reshaped_result
 
+    def fit_transform(self, imgs, y, groups=None):
```

Comment:
adding this rather than rely on the one from the TransformerMixin from sklearn that does not have a meaningful doc string for image inputs